### ISSUE TITLE
Cost per inscription

### ISSUE THREAD

**Issue by mc9625** [ordinals/ord#1807]
I can't understand why I end paying more than expected when I do ord wallet inscribe --fee-rate X <filename>

Let's do some math: I sent 15$ in BTC to the ord wallet.

ord wallet balance results "cardinal": 88307

I inscribe an ordinal: ord wallet inscribe --fee-rate 5 image.webp (6KB)

now 6KB x 5 Sat/Vb = 7500 sats = 1.87 $

the operation reports: "fees":9280

Then I replied the operation a second time:

The operation fees are now: 8300

And now I can't do that a third time. The ord wallet balance is now 8294.

So basically I used 15$ to do an operation that should cost less 2$ twice. 4$ of operation costed 15$ (or so) I would expect to be able to do that operation at least 5 times.

What I am doing wrong?

**Comment by DanielHook15** [comment]
I made the same strange observation. Looking into the block explorer I saw that the transaction fee for an inscription was quite small as expected. But there was also a much larger amount (not a fee) that was transferred to another address. I do not know whose address this is. Perhaps it has something to do with the way how the information of the inscriptions is stored. I hope that someone has understood this and can explain.

**Comment by GaloisField2718** [comment]
Basically what I understood about this is : 
When you make a transaction you have 2 or 3 outputs --> 1 for fees, 1 for receiver and 1 for the rest of change. 
Each transaction function by pack (if I send you 0.1 BTC you will be able to spend 0.05 BTC by spliting this amount in 0.05 BTC for receiver, x BTC fees and 0.05-x BTC will come back to your balance and you will be able to spend this amount in the same way as previous, it's called UTXO, unspent transaction output). 
All of this steps are made automatically by our wallets and maybe for another example you got 0.05 BTC and 0.05 BTC if you want to send 0.1 BTC it will send in 1 transaction two inputs (0.05+0.05) and you should pay more fees than if you take only one input. 

In the case of ord, I think there are two reasons : 
1) What I told above, all of your UTXOs was send to Bitcoin and so your wallet doesn't consider the rest (in fact you should see the good amount in bitcoin-core or sparrow wallet).
2) `ord` seems to not integrate every change address and this cause me in fact many issues that I'm trying to solve. 

I don't know if my explanation is clear and helpful, I hope so ^^

**Comment by GaloisField2718** [comment]
And to answer to the question 
>I would expect to be able to do that operation at least 5 times

You need to send to your ord wallet 5 transactions about ~$4 for each. Now you will have 4 UTXO which could be used by each of your inscriptions.
